src/ui/components/album.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+
+
+function popUpImage(id, name, e) {
```

Comment:
You are mixing imperative UI with declarative UI here - this would be better done by having state somewhere for which image is "active", and then rendering the modal with that image when it's not null. The style updates should be handled with toggling classes.